### PR TITLE
Consider 'not yet' as most recent when sorting running tests by 'Started'

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/57413

### CODE REVIEW

File: assets/javascripts/tests.js [hunk 1]
```diff
@@ -120,9 +120,8 @@ function renderTimeAgo(data, type, row, position, notAvailableMessage) {
         return (haveData ?
             ('<span title="' + data + '">' + jQuery.timeago(data) + '</span>') :
             (notAvailableMessage ? notAvailableMessage : 'not yet'));
-    } else {
-        return haveData ? data : 0;
     }
+    return haveData ? data : 'Z-9999-12-31';
```
Comment (coolo) [inline assets/javascripts/tests.js]: a Y10K bug!!

Comment (coolo) [reply assets/javascripts/tests.js]: (you might be too young to know the reference ;)

Comment (Martchus) [reply assets/javascripts/tests.js]: I actually know about this. But I guess in year 10 000 this is no longer relevant :-)

Comment (coolo) [reply assets/javascripts/tests.js]: that's what the cobol developers in the 60ties thought as well :)

Comment (Martchus) [reply assets/javascripts/tests.js]: I could add an entry here: https://en.wikipedia.org/wiki/Timeline_of_the_far_future#Technological_projects :-)

Comment (Martchus) [reply assets/javascripts/tests.js]: And according to the same wiki article "The Gregorian calendar will have drifted by about 10 days in relation to the seasons". So the calendar system would likely need to be adapted anyways.

**Pull request merged**